### PR TITLE
Update chalk dependency

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

As part of prepping this repo to merge into terra-toolkit. I'm going through and updating dependencies.

### Additional Details

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const defaults = {
 };
 
 function DuplicatePackageCheckerPlugin(options) {
-  this.options = Object.assign({}, defaults, options);
```
Comment (mjhenkes) [inline lib/index.js:16]: this was auto updated by lint

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Changelog
 Unreleased
 ----------
 
+* Update version of chalk used.
```
Comment (StephenEsser) [inline CHANGELOG.md]: Does this need a section? Changed?

Comment (mjhenkes) [reply CHANGELOG.md]:
```suggestion
* Changed
  * Update version of chalk used.
```

File: tests/jest/non-root-package-json/__snapshots__/index.test.js.snap [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 exports[`Dependency tree with non-root package.json should output warnings 1`] = `
 "a
-  Multiple versions of a found:
-    1.0.0 ./~/a
-    2.0.0 ./~/b/~/a
+[0m  Multiple versions of [0m[32m[1ma[22m[39m[37m found:[39m
+[37m[39m    [32m[1m1.0.0[22m[39m [37m[1m./~/a[22m[39m
+    [32m[1m2.0.0[22m[39m [37m[1m./~/b/~/a[22m[39m
```
Comment (StephenEsser) [inline tests/jest/non-root-package-json/__snapshots__/index.test.js.snap:7]: Are these snapshots being written how you would expect with this update? This is a little unreadable, are these special charters for colors/formatting?

Comment (mjhenkes) [reply tests/jest/non-root-package-json/__snapshots__/index.test.js.snap:7]: yeah, thats the color formatting for the terminal output.

**Review by StephenEsser — APPROVED** [review]

File: tests/jest/non-root-package-json/__snapshots__/index.test.js.snap (hunk 3 above)
Comment (mjhenkes) [reply tests/jest/non-root-package-json/__snapshots__/index.test.js.snap:7]: If you want to dig in deeper they're ansi escape codes https://en.wikipedia.org/wiki/ANSI_escape_code
I think at on point is was trying to track down those codes for some reason and they were really hard to google :)

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**